specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/should.not.escape.binary.frame/request.rpt

```diff
@@ -0,0 +1,39 @@
+property key ${ws:handshakeKey()}
+property writeMask ${ws:randomBytes(4)}
+
+connect http://localhost:8001/echo
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Key" ${key}
+write header "Sec-WebSocket-Version" "13"
+write header "Sec-WebSocket-Extensions" "x-kaazing-ping-pong"
+
+read status "101" /.+/
+read version "HTTP/1.1"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
+# The 4 control bytes for ping-pong extension are 0x01010102:
```

Comment:
The request script should not be assuming what the control bytes are. This is a defect in the other tests to, which I did not pick up on when I originally merged the ping pong spec work. Instead the request scripts should use a function to read the control bytes from the initial handshake and use them afterwards. I will submit a pull request with this change for this script which you can merge into your branch. Please then go ahead and change the other request.rpt scripts to use the same system to avoid assuming 01010102. Note that the response scripts do not need to be changed, since by spec they are free to pick any valid value for control bytes.